frontend/benefit/applicant/src/components/applications/forms/application/step3/useApplicationFormStep3.ts

```diff
@@ -21,6 +23,38 @@ const useApplicationFormStep3 = (
   const handleNext = (): void => onNext(application);
   const handleSave = (): void => onSave(application);
 
+  const isRequiredAttachmentsUploaded = (): boolean => {
+    if (
+      application.benefitType === BENEFIT_TYPES.EMPLOYMENT ||
+      application.benefitType === BENEFIT_TYPES.SALARY
+    ) {
+      const hasWorkContract = !isEmpty(
+        application?.attachments?.find(
+          (att) => att.attachmentType === ATTACHMENT_TYPES.EMPLOYMENT_CONTRACT
+        )
+      );
+      let hasPaySubsidyDecision = true;
+      if (application.paySubsidyGranted) {
+        hasPaySubsidyDecision = !isEmpty(
+          application?.attachments?.find(
+            (att) =>
+              att.attachmentType === ATTACHMENT_TYPES.PAY_SUBSIDY_CONTRACT
+          )
+        );
+      }
```

Comment:
Does this shorter version do the trick? :D 
```suggestion
      const hasPaySubsidyDecision = application.paySubsidyGranted && !isEmpty(
        application.attachments?.find(
          (att) =>
            att.attachmentType === ATTACHMENT_TYPES.PAY_SUBSIDY_CONTRACT
        )
      );
```

Reply:
or just 
```
const hasPaySubsidyDecision = application.paySubsidyGranted && !application.attachments?.find(
          (att) =>
            att.attachmentType === ATTACHMENT_TYPES.PAY_SUBSIDY_CONTRACT
      );
      ```